SPECS/python-geomet/python-geomet.spec

```diff
@@ -0,0 +1,64 @@
+%global srcname geomet
+%global debug_package %{nil}
+
+Name:           python-%{srcname}
+Version:        0.2.1
+Release:        1%{?dist}
+Summary:        GeoJSON <-> WKT/WKB conversion utilities
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Languages/Python
+Url:            https://github.com/geomet
+Source0:        https://github.com/geomet/geomet/archive/refs/tags/%{version}.tar.gz#/%{srcname}-%{version}.tar.gz
+BuildArch:      noarch
+
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+%if %{with_check}
+BuildRequires:  python3-pip
+%endif
+
+%global _description \
+Convert GeoJSON to WKT/WKB (Well-Known Text/Binary), and vice versa.
+
+%description %_description
+
+%package -n python3-%{srcname}
+Summary:        %{summary}
+Requires:       python3
+Requires:       python3-six
+Requires:       python3-click
+Requires:       python3-setuptools
```

Comment:
I believe `python3-setuptools` is only a build-time requirement, given upstream's [requirements](https://github.com/geomet/geomet/blob/master/requirements.txt) file.

Reply:
Done